assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java

```diff
@@ -971,6 +965,64 @@ private void configureScale(int scale, Locale locale) {
         nf.setGroupingUsed(false);
         when(formattedText.getNumberFormat(dec, dec, false)).thenReturn(nf);
     }
+    
+	@Test
```

Comment:
Nice job adding a unit test 👍 

We know this is a test method because of the `@Test` annotation please try and create short descriptive method names, how about `mergeAssignmentFromXML`

Reply:
Changed name to mergeAssignmentFromXML